metricbeat/modules.d/traefik.yml.disabled

```diff
@@ -0,0 +1,7 @@
+# Module: traefik
+# Docs: https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-module-traefik.html
```

Comment:
I like the link to the docs. One issue this has is that we `hardcode` master here to every released config file will point to master. Would be nice if in the future this would be a variable that is replaced by the version during packaging.